ReactiveCocoa/Swift/Property.swift

```diff
@@ -97,6 +109,16 @@ public final class MutableProperty<Value>: MutablePropertyType {
 		}
 	}
 
+	/// A signal that will send the property's changes over time,
+	/// then complete when the property has deinitialized.
+	public lazy var signal: Signal<Value, NoError> = { [unowned self] in
```

Comment:
Yeah. Feel free to extract that now, but I think we should leave it as `internal` for now.